src/pylintrc

```diff
@@ -0,0 +1,36 @@
+[MESSAGES CONTROL]
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once).You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+disable=import-error,wrong-import-position
```

Comment:
Do any of these exist in our source any more?  I think I got rid of all of them with apricot.

But should they be ignored otherwise?  Are they not usually pretty straightforward to fix?